epam-electric/components/widgets/Badge.tsx

```diff
@@ -0,0 +1,15 @@
+import { createSkinComponent } from '@epam/uui-core';
+import { Badge as UuiBadge, BadgeFill as UuiBadgeFill, BadgeSize as UuiBadgeSize, BadgeColor as UuiBadgeColor, BadgeCoreProps } from '@epam/uui';
+
+export type BadgeMods = {
+    /** @default 'neutral' */
+    color?: UuiBadgeColor | 'yellow'| 'orange' | 'purple' | 'cyan' | 'mint' | 'white' | 'night100' | 'night600';
+    /** @default 'solid' */
+    fill?: UuiBadgeFill;
+    /** @default '36' */
+    size?: UuiBadgeSize;
+};
+
+export type BadgeProps = BadgeCoreProps & BadgeMods;
+
+export const Badge = createSkinComponent<BadgeCoreProps, BadgeProps>(UuiBadge);
```

Comment:
I first generic you need to use interface of full props from uui - uuiBadgeProps, not only BadgeCoreProps, because in such case you will lose mods from UUI.

Reply:
fixed